app/models/api_docs/service.rb

```diff
@@ -93,6 +95,12 @@ def specification
   end
 
   private
+
+  def service_belongs_to_account
+    return true if (my_service = (service || Service.find_by(id: service_id))) && my_service.account_id == account_id
```

Comment:
Cosmetic, but can
```ruby
my_service = service || Service.find_by(id: service_id)
```
be in a separate line?

Reply:
right, it will be more easily readable 😉